app/Transformers/UserTransformer.php

```diff
@@ -137,6 +139,20 @@ public function includeRecentActivities(User $user)
         );
     }
 
+    public function includeMostPlayedBeatmapsetCount(User $user)
```

Comment:
I don't think this is still used? Probably want to remove it because it's still being included and running the queries...